Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
Hmm, I guess then I'd vote to remove the marker and rename them every time. Otherwise developers can run in staleness issues when others make changes to the fixtures directory.